source/community.applications/usr/local/emhttp/plugins/community.applications/scripts/dockerConvert.php

```
@@ -23,149 +25,194 @@
 <script>
-  <? if ( $json ):?>
-    window.parent.location = "/Apps/AddContainer?xmlTemplate=default:<?=$caPaths['dockerSearchInstall']?>";
+	<? if ( $json ):?>
+		window.parent.location = "/Apps/AddContainer?xmlTemplate=default:<?=CA_PATHS['dockerSearchInstall']?>";
 
-  <? else:?>
-    alert("<?=$error?>");
-    window.parent.location = "/Apps/AddContainer?xmlTemplate=default:<?=$caPaths['dockerSearchInstall']?>";
+	<? else:?>
+		alert("<?=$error?>");
+		window.parent.location = "/Apps/AddContainer?xmlTemplate=default:<?=CA_PATHS['dockerSearchInstall']?>";
```

PHPStan (2.1.51)

[error] Constant CA_PATHS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)